HW2/LengthOfLastWord.java

```diff
@@ -0,0 +1,16 @@
+package HW2;
+
+public class LengthOfLastWord {
+    public static void main(String[] args) {
+        lengthOfLastWord("hello test word");
```

Comment:
incorrect result with input "   fly me   to   the moon  "

Reply:
Trim string added to fix incorrect result -committed and pushed